externs/shaka/manifest.js

```diff
@@ -503,7 +503,8 @@ shaka.extern.FetchCryptoKeysFunction;
  *   A map containing the description of closed captions, with the caption
  *   channel number (CC1 | CC2 | CC3 | CC4) as the key and the language code
  *   as the value. If the channel number is not provided by the description,
- *   we'll set an 0-based index as the key.
+ *   we'll set an 0-based index as the key. If the language code is not
```

Comment:
Nit: (I know this isn't your typo but I just noticed it) "an" -> "a"

Reply:
Done!